build-system/runner/test/org/ampproject/TransformTest.java

```diff
@@ -0,0 +1,63 @@
+package org.ampproject;
+
+import java.util.Set;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import com.google.javascript.jscomp.Compiler;
+import com.google.javascript.jscomp.CompilerPass;
+import com.google.javascript.jscomp.CompilerTestCase;
+import com.google.javascript.rhino.Node;
+
+
+/**
+ * Tests {@link AmpPass}.
+ */
+public class TransformTest extends CompilerTestCase {
+
+  ImmutableMap<String, Set<String>> suffixTypes = ImmutableMap.of();
+
+  ImmutableMap<String, Node> assignmentReplacements = ImmutableMap.of();
+
+  ImmutableMap<String, Node> prodAssignmentReplacements = ImmutableMap.of();
+  
+  ImmutableSet<String> blacklist = ImmutableSet.of();
+  
+  public TransformTest() {
+    super(DEFAULT_EXTERNS);
+    parseTypeInfo = true;
+    runTypeCheckAfterProcessing = true;
+  }
+
+  @Override
+  public void setUp() throws Exception {
+    super.setUp();
+    disableTypeCheck();
+  }
+
+
+  @Override protected CompilerPass getProcessor(Compiler compiler) {
+    return new AmpPass(compiler, /* isProd */ true, suffixTypes, assignmentReplacements,
+        prodAssignmentReplacements, blacklist);
+  }
+
+  @Override protected int getNumRepetitions() {
+    // This pass only runs once.
+    return 1;
+  }
+
+  public void testProxyTransformSimple() throws Exception {
+    enableTypeCheck();
```

Comment:
@cramforce do you have any idea why this doesn't work? I've been trying to follow the WizPassTest.java code but running into

```
 [junit] Testcase: testProxyTransformSimple(org.ampproject.TransformTest):	Caused an ERROR
    [junit] com.google.javascript.rhino.Node.<init>(I)V
    [junit] java.lang.NoSuchMethodError: com.google.javascript.rhino.Node.<init>(I)V
    [junit] 	at com.google.javascript.rhino.jstype.FunctionParamBuilder.<init>(FunctionParamBuilder.java:52)
    [junit] 	at com.google.javascript.rhino.jstype.JSTypeRegistry.createOptionalParameters(JSTypeRegistry.java:1264)
    [junit] 	at com.google.javascript.rhino.jstype.JSTypeRegistry.initializeBuiltInTypes(JSTypeRegistry.java:302)
    [junit] 	at com.google.javascript.rhino.jstype.JSTypeRegistry.resetForTypeCheck(JSTypeRegistry.java:255)
    [junit] 	at com.google.javascript.rhino.jstype.JSTypeRegistry.<init>(JSTypeRegistry.java:195)
    [junit] 	at com.google.javascript.jscomp.Compiler.getTypeRegistry(Compiler.java:1213)
    [junit] 	at com.google.javascript.jscomp.CompilerTestCase.test(CompilerTestCase.java:766)
    [junit] 	at com.google.javascript.jscomp.CompilerTestCase.test(CompilerTestCase.java:725)
    [junit] 	at com.google.javascript.jscomp.CompilerTestCase.test(CompilerTestCase.java:658)
    [junit] 	at com.google.javascript.jscomp.CompilerTestCase.test(CompilerTestCase.java:564)
    [junit] 	at org.ampproject.TransformTest.testProxyTransformSimple(TransformTest.java:51)
```